addons/packages/ako-operator/1.6.0/bundle/.imgpkg/images.yml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: imgpkg.carvel.dev/v1alpha1
+images:
+- annotations:
+    kbld.carvel.dev/id: projects-stg.registry.vmware.com/tkg/ako-operator:v1.5.0_vmware.1
```

Comment:
this should be changed later (in another PR), after we have formal 1.6.0 ako-o build

Reply:
ok